Configurations/10-main.conf

```diff
@@ -1594,29 +1588,29 @@ sub vms_info {
         inherit_from     => [ "darwin-common", asm("ppc32_asm") ],
         cflags           => add("-arch ppc -std=gnu9x -DB_ENDIAN -Wa,-force_cpusubtype_ALL"),
         perlasm_scheme   => "osx32",
-        shared_ldflag    => "-arch ppc -dynamiclib",
+        shared_ldflag    => "-arch ppc",
```

Comment:
Originally shared libraries were linked without `$(CFLAGS)`. And this was the reason for why one had to have things like -arch/-xarch/-m64 in ldflags. Now they are linked with `$(CFLAGS)` that already has -arch/-xarch/-m64 and so defining them in ldflags is actually redundant. In other words if -dynamic link is no longer needed, then one can as well just remove this line. As well as corresponding lines in all `darwin*` targets.

Reply:
Good point